### ISSUE TITLE
EnvPlugin: The global process is not available after hot-reload

### ISSUE THREAD

**Issue by gerhardsletten** [fuse-box/fuse-box#159]
With EnvPlugin, the global var process is not available after a hot-reload, you need to do a import for it to be:

```js
import process from 'process'
console.log(process.env.someVar)
```

**Comment by nchanged** [comment]
Sorry for moving that one, we are on fire now. Will be fixed soon enough.

**Comment by arabsight** [comment]
Hi, @nchanged  this is still happening in 1.4.1-beta-9

**Comment by nchanged** [comment]
Hi! Is that so? Can u repro it?

**Comment by arabsight** [comment]
Hi, here you go: https://github.com/arabsight/fusebox-env-issue
first build works (this is injected: **var process = require("process");**), but when you modify something it gets removed.

**Comment by nchanged** [comment]
thanks, i will check it!

**Comment by devmondo** [comment]
this should be fixed in version 2.0 let us know.

we will close this for now, but feel free to open it otherwise :)
